packages/manager/modules/emailpro/src/domain/emailpro-domain.html

```diff
@@ -251,6 +251,36 @@
                                         data-simplepopover-placement="top"
                                         data-simplepopover-single="true"
                                     ></button>
+
+                                    <!-- dkim -->
+                                    <button
+                                        class="oui-badge oui-badge_success"
+                                        data-translate="emailpro_tab_domain_diagnostic_dkim"
+                                        data-ng-if=":: dkimGlobalStatus(domain) === GLOBAL_DKIM_STATUS.OK && !exchange.isMXPlan"
+                                        disabled
```

Comment:
**question:** Should this button be disabled always?

Reply:
Yes if we have badge_success (if the configuration is good there isn't action onclick).